parser/parser.mly

```diff
@@ -119,7 +127,7 @@ line            : ID COLON term DOT
                 { mk_rules (List.map scope_rule $1) }
                 | command DOT { $1 }
                 | EOF
-                { mk_ending () ; raise Lexer.EndOfFile }
+                { mk_ending ; raise Lexer.EndOfFile }
```

Comment:
`{ mk_ending () ; raise Lexer.EndOfFile }` was fine.

Reply:
Or maybe you could just inline all these intermediate functions...
`mk_ending ()` is just as readable as `mk_entry(Ending)`.
`mk_rules rs` just as readable as `mk_entry(Rules(rs))`.
And so on...